seeds/companies.js

```diff
@@ -0,0 +1,28 @@
+exports.seed = function(knex) {
```

Comment:
Might be more useful to use [faker](
https://github.com/marak/Faker.js/) for this imo.